src/utils/git.js

```diff
@@ -91,37 +92,48 @@ export async function signIntoGithub(username, password) {
         credentials.token = token.token;
         credentials.url = token.url;
         credentials.isTwoFactorAuth = true;
+        log.info('Successfully logged into GitHub!');
         return credentials;
       } catch (error) {
         log.error('failed to sign into github', error);
       }
     } else if (err.status === 401) {
       // the user's request could not be authenticated, so return false.
       return false;
+    } else if (err.status === 422) {
+      log.error('A github token has already been created.  Please go to GitHub and delete the hammer-to token' +
```

Comment:
would it be easier to just provide a link to the tokens?

https://github.com/settings/tokens